docs/semgrep-code/pro-rules.md

```diff
@@ -8,95 +8,67 @@ tags:
+### Change rule modes or disable Pro rules in Semgrep AppSec Platform
 
-### Adding Pro rules to your Semgrep org through the Registry
+Like any other rule or ruleset, you can disable Pro rules or change their rule mode to leave comments to developers or potentially block a PR.
 
 1. Sign in to [Semgrep AppSec Platform](https://semgrep.dev/login).
-2. Go to [Semgrep Registry](https://semgrep.dev/r).
-3. Click on **Visibility > Pro rules**.
-4. Optional: Apply additional filters by entering search terms in the search box or selecting filters from drop-down boxes.
-5. To add a specific rule or ruleset:
-    1. Click on the card for the **rule or ruleset > Add to Policy**.
-    2. Select a Policy rule mode to add the rules to. It is recommended to start with the Monitor mode to silently gather findings and audit the rule's performance.
-
-### Filtering behavior
-
-* Filter types such as Language and Technology use AND logic. This means that search terms must match all filters. For example, selecting Java (a Language) and security (a Category) shows only rules with both properties (Java and security).
-* Adding filters of the same type use OR logic. This means that search terms can match any of the filters for that type. For example, selecting Java and Python (both languages) shows rules with either language.
-* A gem icon (💎) denotes Semgrep Pro rules.
-
-<!-- Future feature: Add Semgrep Pro rules through your Policy -->
+1. Navigate to **Rules > Policies**.
+1. Under **Source**, click **Pro <i class="fa-solid fa-gem"></i>** to view all the Semgrep Pro rules currently in your Policies.
+1. Find and select the rules you want to disable or change.
+1. Click **Change modes** and select one of the provided options.
 
-<!-- Future feature: Receiving updates on Semgrep Pro rules -->
-
-## Removing or disabling Semgrep Pro rules
+You can find all Semgrep Pro rules in your Policies page, so if you want to re-enable Pro rules, you can simply use the **Source > Pro <i class="fa-solid fa-gem"></i>** filter.
 
-### Disabling rules
+### Add Semgrep Pro rules in CLI or CI
 
-<DisableRule />
+:::info Prerequisites
+For CLI users: You must be [logged in](/getting-started/cli#log-in-to-your-semgrep-account).
+:::
 
-### Removing rulesets
+In some cases, you may want to run a scan with a specific set of Pro rules:
 
-<RemoveRuleset />
+1. Go to [Semgrep Registry](https://semgrep.dev/r).
+2. Click **Visibility > Pro rules**.
+3. Optional: Apply additional filters by entering search terms in the search box or selecting filters from drop-down boxes.
+4. For a single rule, click the **Rule's card > Run locally**. For rulesets, click the card.
+5. Copy and paste the snippet to your CLI or CI configuration file. You can add several rulesets. Snippets can look like these:
+    ```bash
+    semgrep --config "p/c"
+    semgrep --config "p/xss"

```

Comment:
I don't think these are good examples here - most of the rulesets you'd add will have a mix of OSS and Pro rules, and I don't think even filtering the Registry to Pro before you browse would change that. I'm not actually sure what a good example would be, unfortunately, but it might be worth checking with the Code folks.

Reply:
Opted to use a screenshot instead to make sure the user doesn't get lost.